drape_frontend/apply_feature_functors.cpp

```diff
@@ -116,15 +107,15 @@ class LinesStat
 private:
   LinesStat() = default;
 
-  using TKey = pair<FeatureID, int>;
+  using TKey = std::pair<FeatureID, int>;
   struct TValue
```

Comment:
TReadAndNeededPoints or use std::pair and give meaningful names in the loop using structured binding.

Reply:
I've fixed more "technical" of your notes, but I see no point in a bigger renaming/refactoring of a debug-only code used once a year (hopefully). My intention was just to fix it so that it works again :)